controllers/wizard/fileUpload/FileUploadWizardHandler.php

```diff
@@ -415,7 +415,7 @@ public function uploadFile($args, $request)
 
         // Validate the form and upload the file.
         if (!$uploadForm->validate()) {
-            return new JSONMessage(false, $uploadForm->fetch($request));
+            return new JSONMessage(true, $uploadForm->fetch($request));
```

Comment:
I think this doesn't make sense, everywhere else is using `JSONMessage(false)` for failures 🤔

Reply:
Thats what I thought as well initially. But JSONMessage(false, 'error') these are errors that are supposed to be communicated via alert. But Form validations works differently, where you refetch the form so it can show updated (validation error state). If you search for `->validate()` you will see this pattern all over.

Reply:
Yep! I've just seen it, I'll change my decision to approved. I don't think it's worth to fix it properly, this code will eventually disappear.